R/utils.R

```diff
@@ -143,3 +143,28 @@ summarise_commit <- function(object) {
 
   return(out)
 }
+
+#' @title Load an application and instructions to run shiny.benchmark
+#'
+#' @param path A character vector of full path names
+#'
+#' @importFrom glue glue
+#' @export
+load_example <- function(path) {
+  # see if path exists
+  if (!file.exists(path))
+    stop("You must provide a valid path")
+
+  ex_path <- system.file(
+    "examples",
+    package = "shiny.benchmark",
+    mustWork = TRUE
+  )
+  files <- list.files(path = ex_path, full.names = TRUE)
+
+  invisible(
```

Comment:
It would be great to make this more verbose. This is an example, so the user should know what is happening - I'm ok even with adding a message for each copied file.

Reply:
Make sense. I will add it as well.